test/e2e-go/cli/goal/expect/goalExpectCommon.exp

```diff
@@ -565,14 +614,27 @@ proc ::AlgorandGoal::SignTransaction { WALLET_NAME WALLET_PASSWORD INPUT_TXN OUT
     }
 }
 
+# Transfer asset, rekey
+proc ::AlgorandGoal::SignTransactionRekey { WALLET_NAME REKEY_ACCOUNT_ADDRESS INPUT_TXN OUTPUT_TXN TEST_PRIMARY_NODE_DIR} {
```

Comment:
Note that this isn't optimal as it's the same as the `SignTransaction` procedure plus an additional argument.  It's good enough for now, but this should be revisited after the release.